.taskcluster.yml

```diff
@@ -307,7 +306,7 @@ tasks:
 
       - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
         then:
-          taskId: {$eval: as_slugid("strip_windows_binary")}
```

Comment:
We could drop this task now and just generate the zipin the `build_windows_release` task.
If you don't want to do it now, please file an issue and add it to the CI project (see, nothing is ever complete :P).

Reply:
> (see, nothing is ever complete :P).

(for now :P)

Reply:
#430

Reply:
#430 is about reducing the size of the binary, we should also file an issue about dropping the `strip` task (which is no longer needed, and might not be required for #430 either).

Reply:
It could be required for #430, since it will be much simpler to install software that might reduce the size of the binaries in a Linux image instead of a Windows one.

Reply:
All the suggestions from the links in #430 refer to things to change which are either in the Rust source code or in the Cargo.toml and don't require external tools.

Reply:
You're right, updated the issue, forgot a link :P